src/rules.js

```diff
@@ -2272,7 +2272,7 @@ module.exports = [
         "matcher": "Matrix3d",
         "allowParamToValue": false,
         "styles": {
-            "transform": "matrix($0)"
+            "transform": "matrix($0,$1,$2,$3,$4,$5,$6,$7,$8,$9,$10,$11,$12,$13,$14,$15)"
```

Comment:
I originally coded like that as well but I realized this is an edge case and it's better to be defined in the config instead of being in the class name. Same to `matrix`. `allowParamToValue` should be false for both.